lib/matplotlib/backends/backend_qt.py

```diff
@@ -585,6 +585,12 @@ def resize(self, width, height):
         self.canvas.resize(width, height)
         self.window.resize(width + extra_width, height + extra_height)
 
+    @classmethod
+    def start_main_loop(cls):
+        qapp = QtWidgets.QApplication.instance()
```

Comment:
We now do the "does the app exist" dance in `__init__`, but this is a static method so do we want to use `_creatApp` here as well?

Reply:
or at least bail if it does not exist.

Reply:
sure